### PR TITLE
add note

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3351

### CODE REVIEW

File: migration/sharepoint-migration-supported-list-templates.md [hunk 1]
```diff
@@ -96,3 +96,6 @@ The following table details SharePoint List templates and which ones are support
 |1220|HealthRules|Health Rules. Value = 1220|No|
 |1221|HealthReports|Health Reports. Value = 1221|No|
 |1230|DeveloperSiteDraftApps|Draft Apps library in Developer Site. Value = 1230|No|
+
+>[!NOTE]
+>Sites Pages (for example Wiki Page Library) migration is available only when the source selected is _all lists, libraries, pages, and web parts_ as SPMT treats them as part of the whole site.
```
Comment (JohanFreelancer9) [inline migration/sharepoint-migration-supported-list-templates.md:101]:
```suggestion
> [!NOTE]
> Sites Pages (for example Wiki Page Library) migration is available only when the source selected is _all lists, libraries, pages, and web parts_ as SPMT treats them as part of the whole site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by get-itips** [comment]
@JoanneHendrickson please review / merge. Thanks

**Comment by get-itips** [comment]
@JoanneHendrickson please help me with review and merge

**Pull request merged**